shipstation/api.py

```diff
@@ -202,43 +205,41 @@ def list_services(
             for service in services.json(parse_float=Decimal)
         ]
 
-    def get_customer(
-        self, customer_id: str
-    ) -> typing.Union[str, ShipStationBase, None]:
+    def get_customer(self, customer_id: str) -> str | ShipStationBase | None:
         customer = self.get(endpoint=f"/customers/{customer_id}")
         return ShipStationCustomer().json(customer.text)
 
-    def list_customers(self, parameters: typing.Any = {}) -> Page:
+    def list_customers(self, parameters: Any = {}) -> Page:
         valid_parameters = self._validate_parameters(
             parameters, CUSTOMER_LIST_PARAMETERS
         )
         return Page(
             type=ShipStationCustomer,
             key="customers",
-            params=parameters,
-            call=(self.get, {"endpoint": "/customers", "payload": valid_parameters}),
+            params=valid_parameters,
+            call=(self.get, {"endpoint": "/customers"}),
         )
 
-    def list_fulfillments(self, parameters: typing.Any = {}) -> Page:
+    def list_fulfillments(self, parameters: Any = {}) -> Page:
         valid_parameters = self._validate_parameters(
             parameters, FULFILLMENT_LIST_PARAMETERS
         )
         return Page(
             type=ShipStationFulfillment,
             key="fulfillments",
-            params=parameters,
-            call=(self.get, {"endpoint": "/fulfillments", "payload": valid_parameters}),
+            params=valid_parameters,
+            call=(self.get, {"endpoint": "/fulfillments"}),
```

Comment:
@agritheory I've removed the `payload` option if params are provided since they are overridden in the pagination logic:

https://github.com/agritheory/shipstation-client/blob/62b729288c31ade546e3c50c66dfab7ed115c15c/shipstation/pagination.py#L23-L28